src/util/elias_fano_iter.rs

```diff
@@ -81,14 +81,12 @@ macro_rules! gen_ef_iter_impl {
 
             /// Returns the number of elements that this iterator will iterate over. The size is
             /// precise.
-            #[must_use]
```

Comment:
A lot of the must_use got removed here; is this an accident?

Reply:
No, those are trait implementations. #[must_use] is meaningless in trait implementations (as the parent trait has to define it), and in nightly they cause compiler warnings now.